src/librbd/image/OpenRequest.cc

```diff
@@ -78,6 +78,9 @@ Context *OpenRequest<I>::handle_v1_detect_header(int *result) {
     }
     send_close_image(*result);
   } else {
+    lderr(cct) << "RBD image format 1 is deprecated. "
+               << "Please migrate this image to image format 2." << dendl;
```

Comment:
maybe say "copy" instead of "migrate" so there's no confusion about how to do it (this was a common question when format 2 came out)